vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -260,4 +262,8 @@ void TextViewManager::OnDescendantTextPropertyChanged(ShadowNodeBase *node) {
   }
 }
 
+TextTransform TextViewManager::GetTextTransformValue(ShadowNodeBase *node) {
+  return static_cast<TextShadowNode *>(node)->textTransform;
```

Comment:
Not super happy about this, but I didn't want to pull TextShadowNode into the header. I'll at least add a safety check to make sure the shadow node is actually from RCTText

Reply:
The alternative to this approach was to force a full textTransform update from the root, which in practice is probably not very expensive, but this allows us to apply the correct inherited textTransform directly to a raw text node that changed (without needing to recurse the entire text tree).

Reply:
sounds like an assert that the view manager is named Text or whatever it's called should be enough at least to catch things in debug :)
Ideally we have some level of rtti on these things without actually using rtti

Reply:
This has been fixed with the view manager assertion.